chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -1606,6 +1606,50 @@ impl pallet_multiblock_migrations::Config for Runtime {
     type WeightInfo = weights::pallet_multiblock_migrations::SubstrateWeight<Runtime>;
 }
 
+/// Maintenance mode Call filter
+pub struct MaintenanceFilter;
+impl Contains<RuntimeCall> for MaintenanceFilter {
+    fn contains(c: &RuntimeCall) -> bool {
+        !matches!(
+            c,
+            RuntimeCall::Balances(..)
+                | RuntimeCall::Registrar(..)
+                | RuntimeCall::Session(..)
+                | RuntimeCall::System(..)
+                | RuntimeCall::PooledStaking(..)
+                | RuntimeCall::Utility(..)
+                | RuntimeCall::XcmPallet(..)
+                | RuntimeCall::EthereumBeaconClient(..)
+                | RuntimeCall::EthereumSystem(..)
+                | RuntimeCall::EthereumTokenTransfers(..)
+                | RuntimeCall::ContainerRegistrar(..)
+                | RuntimeCall::ServicesPayment(..)
+                | RuntimeCall::DataPreservers(..)
+                | RuntimeCall::Hrmp(..)
+                | RuntimeCall::Proxy(..)
+                | RuntimeCall::Multisig(..)
+                | RuntimeCall::AssetRate(..)
+                | RuntimeCall::StreamPayment(..)
+        )
+    }
+}
+
+/// Normal Call Filter
+pub struct NormalFilter;
+impl Contains<RuntimeCall> for NormalFilter {
+    fn contains(c: &RuntimeCall) -> bool {
+        !IsRelayRegister::contains(c) || !IsParathreadRegistrar::contains(c)
```

Comment:
```suggestion
        !IsRelayRegister::contains(c) && !IsParathreadRegistrar::contains(c)
```

Don't we have tests?

Reply:
Yeah some of the tests are failing. I added the OR clause because the call cannot be a `RelayRegister` and a `ParathreadRegistrar` at the same time, but seems to be failing for some reason, so I will add the previous `EverythingBut` to see if the error is there.